defaults/main.yml

```diff
@@ -10,10 +10,10 @@ storage_pool_defaults:
   type: lvm
 
   encryption: false
-  encryption_passphrase: null
-  encryption_key_file: null
+  private_passphrase: null
+  private_key: null
   encryption_cipher: null
-  encryption_key_size: null
+  private_key_size: null
```

Comment:
I am pretty sure that this key size corresponds with the LUKS master key size and not the binary blob specified in `private_key`.

Reply:
Basically LUKS uses one or more passphrases/binary blobs that are used to encrypt the LUKS master key. The result is stored in key slots. The master key is what is actually used to encrypt the blocks on the storage device. Not sure which details changed with LUKS2 but I guess the general concept stayed the same.

Reply:
reverted